coc/client.py

```diff
@@ -278,14 +278,19 @@ def login_with_keys(self, *keys: str) -> None:
 
         LOG.debug("HTTP connection created. Client is ready for use.")
 
-    def close(self) -> None:
+    def close_loops(self) -> None:
         """Closes the HTTP connection
         """
         LOG.info("Clash of Clans client logging out...")
         self.dispatch("on_client_close")
         self.loop.run_until_complete(self.http.close())
         self.loop.close()
```

Comment:
```suggestion
    def close_loops(self) -> None:
        """Closes the HTTP connection
        """
        self.loop.run_until_complete(self.http.close())
        self.loop.close()
```

Reply:
> Why are you omitting the dispatch?

because this is actually closing the `asyncio loop` and not doing what the dispatch name says. The thing that dispatch name says is actually being one by the `close_client` method